javascript/ql/src/semmle/javascript/TypeScript.qll

```diff
@@ -1098,7 +1098,7 @@ class ConditionalTypeScope extends @conditionaltypescope, Scope {
  * In the above example, `List` is a concrete expression, `string` is a type annotation,
  * and `List<string>` is thus an expression with type arguments.
  */
-class ExpressionWithTypeArguments extends @expressionwithtypearguments, Expr {
+class ExpressionWithTypeArguments extends @expr_ession_with_type_arguments, Expr {
```

Comment:
`expression_with_type_arguments`

or the shorter `expr_with_type_arguments` if you prefer